src/frontend/js/hooks/useMatchMedia.ts

```diff
@@ -34,4 +34,10 @@ const useMatchMedia = (query: string): boolean => {
   return matches;
 };
 
+const emCalc = (px: number) => {
+  return px / 16;
+};
```

Comment:
em change depending of font-size.
```suggestion
const emCalc: (px: number, fontSize?: number) => number = (px, fontSize = 16) => {
  return px / fontSize;
};
```

Reply:
Indeed! But I remove it in a recent commit based on https://github.com/openfun/richie/pull/2355#discussion_r1576402747